### PR TITLE
updates and new screenshots

### PR DESCRIPTION
**Pull request by ma-zah**

### CODE REVIEW

File: changelog.mdx [hunk 1]
```diff
@@ -4,6 +4,61 @@ description: "All the nice stuff you asked for"
...
+<Frame caption="Relaunch AI discovery based on your comments, screenshot 11/2023">
+  <img
+    src="images/changelog/comments-screenshot.png"
+    alt="Relaunch AI discovery based on comments"
+  />
+</Frame>
+
+## 2023-11-09
+
+- `In-context screenshots` of executed steps visibile directly in test report detail
```
Comment (Germandrummer92) [inline changelog.mdx:37]:
```suggestion
- `In-context screenshots` of executed steps visibile directly in test report details
```

File: discovery.mdx [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
-## Free prompting
+## AI test case generation
 
 We have released our free prompting interface for everyone to try it out as an alpha release. It will help us fine tune the AI agent and the self-serve experience.
 [Sign up to our app](https://app.octomind.dev/setup/url?utm_source=docs&utm_medium=txt-lnk) and get the first UI test automatically.
 
-<Frame caption="Free prompting window for test case discovery, screenshot 10/2023">
+### Editing your AI discovery
+
+We've also added **human feedback** to the AI test case generation process. Pick the discovered step the AI agent got wrong and have the following steps rediscovered with a simple comment.
```
Comment (Germandrummer92) [inline discovery.mdx:18]:
```suggestion
We've also added **human feedback** to the AI test case generation process. Pick the discovered step the AI agent got wrong and have it and the following steps rediscovered with a simple comment.
```

File: execution-without-ci.mdx [hunk 3]
```diff
@@ -33,7 +33,7 @@ The command will return a response with link to your test report, looking someth
 {"testReportUrl":"https://app.octomind.dev/testreports/<some-id>", ...}
 ```
 
-On the above mentioned link, you can find the test report containing all test results of your execution.
+On the above mentioned link, you can find the test report containing all test results of your execution. You'll be able to rerun all test cases from your terminal OR the app.
```
Comment (Germandrummer92) [inline execution-without-ci.mdx:36]:
```suggestion
On the above mentioned link, you can find the test report containing all test results of your execution. You'll be able to run all test cases from your terminal OR the app.
```

File: images/changelog/progress-feat.gif
Comment (Germandrummer92) [inline images/changelog/progress-feat.gif:1]: should we cut out the initial loading spinner? its confusing ;)

File: integrations/quickstart-GitHub.mdx [hunk 4]
```diff
@@ -46,6 +46,15 @@ We’ll ask you to grant access to your repository to read metadata & read and w
   />
 </Frame>
 
+Previous comments with test results will be collapsed by default. You can click to expand them.
```
Comment (Germandrummer92) [inline integrations/quickstart-GitHub.mdx:49]:
```suggestion
Previous comments by octomind with test results will be collapsed by default. You can click to expand them.
```

**Review by Germandrummer92 — APPROVED** [review]

File: images/changelog/progress-feat.gif
Comment (ma-zah) [reply images/changelog/progress-feat.gif:1]: ah, we can

Comment (ma-zah) [reply images/changelog/progress-feat.gif:1]: this week, ok?

**Pull request merged**